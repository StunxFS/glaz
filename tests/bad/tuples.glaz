fn x() (i32, bool) {
	return (true, 1);
}

fn x2() (i32, i32) { return (1, 2); }
fn x3() (i64, i64) { return x(); }

fn main() {}
