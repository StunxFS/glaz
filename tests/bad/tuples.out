tests/bad/tuples.glaz:2:3: error: expected type `(i32, bool)`, found `(bool, i32)`
     | 
   2 | 	return (true, 1);
     | 	^^^^^^^^^^^^^^^^
     = note: in return of function `x`
tests/bad/tuples.glaz:6:23: error: expected type `(i64, i64)`, found `(i32, bool)`
     | 
   6 | fn x3() (i64, i64) { return x(); }
     |                      ^^^^^^^^^^
     = note: in return of function `x3`
glaz error: could not compile `tuples`, aborting due to 2 previous errors.
